### PR TITLE
[v2][storage] Implement `GetServices` and `GetOperations` in v2 factory adapter

### PR DESCRIPTION
**Pull request by mahadzaryab1**
## Which problem is this PR solving?
- Towards #5079 

## Description of the changes
- This PR implements `GetServices` and `GetOperations` in the v2 `factoryadapter`

## How was this change tested?
- Added unit tests

## Checklist
- [x] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [x] I have signed all commits
- [x] I have added unit tests for the new functionality
- [x] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `yarn lint` and `yarn test`

### CODE REVIEW

File: storage_v2/factoryadapter/reader.go [hunk 1]
```diff
@@ -40,12 +40,26 @@ func (*TraceReader) GetTrace(_ context.Context, _ pcommon.TraceID) (ptrace.Trace
...
-func (*TraceReader) GetOperations(_ context.Context, _ tracestore.OperationQueryParameters) ([]tracestore.Operation, error) {
-	panic("not implemented")
+func (tr *TraceReader) GetOperations(ctx context.Context, query tracestore.OperationQueryParameters) ([]tracestore.Operation, error) {
+	o, err := tr.spanReader.GetOperations(ctx, spanstore.OperationQueryParameters{
+		ServiceName: query.ServiceName,
+		SpanKind:    query.SpanKind,
+	})
+	if err != nil {
+		return nil, err
+	}
```
Comment (yurishkuro) [inline storage_v2/factoryadapter/reader.go:54]: you don't actually need this check (makes it easier to test) - you can still loop through `o` even if it's nil

**Review by yurishkuro — APPROVED** [review]

**Review by yurishkuro — APPROVED** [review]
